src/pages/workspace/companyCards/WorkspaceCompanyCardPageEmptyState.tsx

```diff
@@ -70,6 +71,7 @@ function WorkspaceCompanyCardPageEmptyState({policy}: WithPolicyAndFullscreenLoa
                 illustrationStyle={styles.emptyStateCardIllustration}
                 illustrationContainerStyle={[styles.emptyStateCardIllustrationContainer, styles.justifyContentStart]}
                 titleStyles={styles.textHeadlineH1}
+                isButtonDisabled={workspaceAccountID > CONST.DEFAULT_NUMBER_ID}
```

Comment:
I think this should be the opposite, right? this disables it IF `workspaceAccountID` exists, not if it doesn't.

The [comment](https://github.com/Expensify/App/pull/58599/files#r1999810472) above seems to suggest that this is backwards.

Reply:
https://github.com/Expensify/App/pull/58670 if so